.github/workflows/core.yml

```
@@ -297,7 +298,14 @@ jobs:
           path: core/build
       - run: chmod +x core/build/unix/trezor-emu-core*
       - uses: ./.github/actions/environment
+      - name: Start Tropic model
+        if: ${{ env.TREZOR_MODEL == 'T3W1' }}
+        run: nix-shell --arg fullDeps true --run "cd vendor/ts-tvl && mkdir -p logs && poetry env use 3.12 && poetry install && poetry run model_server tcp -c ../../tests/tropic_model.yml >logs/server.log 2>&1 &"
```

Nit: we depend on ts-tvl in top-level pyproject.toml so following should work fine. Eventually I'd like emu.py to launch Tropic emulator automatically.
```suggestion
        run: nix-shell --arg fullDeps true --run "cd vendor/ts-tvl && mkdir -p logs && uv run model_server tcp -c ../../tests/tropic_model.yml >logs/server.log 2>&1 &"
```

---

> Eventually I'd like emu.py to launch Tropic emulator automatically.

Maybe this is the way to go eventually. Would also solve @andrewkozlik 's dislike for the way we do things now. Basically it would mean... if you use a T3W1 emulator it always includes Tropic, there aren't two ways of doing things. Guess I was just too afraid to disrupt others back when I introduced Tropic, but now that the thing has matured, it should just be on by default. Thanks for mentioning this!